var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -437,6 +437,10 @@ def enable_or_disable(variant, keyword="USE", var=None, newer=False):
             env.set("HIPCUB_PATH", self.spec["hipcub"].prefix)
             env.set("ROCTHRUST_PATH", self.spec["rocthrust"].prefix)
             env.set("ROCTRACER_PATH", self.spec["roctracer-dev"].prefix)
+            env.set("USE_NCCL", True)
+            env.set("USE_PTHREADPOOL", True)
```

Comment:
This is already set below for PyTorch 1.6+

Reply:
Removed USE_NCCL.

Reply:
The above comment was for NCCL, this comment is for PTHREADPOOL

Reply:
Removed PTHREADPOOL as well from the recipe.